src/Sandstorm/Sandstorm.ino

```diff
@@ -145,10 +146,10 @@ void setup() {
     Serial.begin(9600);
     
     // This is how the constructor is able to setup all of the code in each of the classes
-    driving.driveSetup(LEFT_MOTOR, RIGHT_MOTOR);
-    depositing.depositSetup(DEPOSIT_MOTOR, DEPOSIT_VIBRATOR);
-    digging.digSetup(DIGGING_MOTOR, DIGGING_ACTUATOR);
-    camera.cameraSetup(HORIZONTAL_SERVO, VERTICAL_SERVO, ARM_SERVO);
+    driveSetup(LEFT_MOTOR, RIGHT_MOTOR);
+    depositSetup(DEPOSIT_MOTOR, DEPOSIT_VIBRATOR);
+    digSetup(DIGGING_MOTOR, DIGGING_ACTUATOR);
+    cameraSetup(HORIZONTAL_SERVO, VERTICAL_SERVO, ARM_SERVO);
```

Comment:
I would again suggest we store a variable in main instead of doing invisible operations like this but that's just me